api/src/main/resources/messages.properties

```diff
@@ -1667,6 +1667,7 @@ Patient.merge.notPreferred=Not Preferred
 Patient.merge.includeVoided=Include deleted
 Patient.merged=Patients merged successfully
 Patient.merge.patient.NoUnvoidedOrders=Not preferred patient(s) should not have any unvoided orders
+Patient.merge.bothHasActiveOrders=Cannot merge patients. Both {0} and {1} patients have active [{2}] orders.
```

Comment:
An possible error message would be:
Cannot merge patients. Both BAH109245 and BAH239281 patients have active [Drug Order] orders.
I hope that message is clear.
It would be good, If you can tell a better message, If you have any.